licenses/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.2.20 on 2021-05-27 17:12
+# Generated by Django 2.2.20 on 2021-06-21 21:20
```

Comment:
This seems like mostly "lint level" changes, but says it was re-generated. Just a safety check to make sure we want to modify existing migrations.